package.json

```diff
@@ -34,7 +34,7 @@
     "jest": "^20.0.4",
     "nodemon": "1.11.x"
   },
-  "dependencies": {
+  "peerDependencies": {
     "graphql": "^0.10.1"
```

Comment:
@ccuilla You will want to also add `graphql` as a devDependencies as well to allow an simple quick install if someone clones your repo directly and wishes to run tests etc.